api/src/main/java/org/ehcache/config/CacheRuntimeConfiguration.java

```diff
@@ -45,7 +45,7 @@
    * @throws java.lang.IllegalStateException if the listener is already registered
    */
   void registerCacheEventListener(CacheEventListener<? super K, ? super V> listener,
-                                  EventOrdering ordering, EventFiring firing, Set<EventType> forEventTypes);
+                                  EventOrdering ordering, EventFiring firing, EnumSet<EventType> forEventTypes);
```

Comment:
Hehe, got carried away with the `EnumSet` indeed, let me revert that back to a `Set`